zstash/hpss_utils.py

```diff
@@ -87,7 +117,18 @@ def add_files(
             logger.debug("Closing tar archive {}".format(tfname))
             tar.close()
 
-            # Transfer tar archive to HPSS
+            tar_md5: str = tarFileObject.md5()
+            tarFileObject.close()
+            # TODO: This function is called by create.py and update.py,
+            # so, we need to test both `create` and `update`.
+            # TODO: test that tars table can be added to existing zstash database
```

Comment:
`update.py` needs to be updated to create the `tars` table if it doesn't exist. Add a test case for this too.